docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,592 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        Update the (approximate) left matrix profile <code>PL</code> in place by checking the right neighbors of
</pre><p> </p><p>revise the definition as the current description can be misleading. We may say:</p><pre class="ql-syntax" spellcheck="false">Update the (approximate) left matrix profile <code>PL</code> for a chunk of forthcoming subsequences by computing their distance to their left neighbor <code>T[query_idx : query_idx+m]</code>

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done!